web/src/App.tsx

```diff
@@ -1,3 +1,5 @@
+// temporary_disabled_rules
+/* eslint-disable @typescript-eslint/no-unused-vars, react-hooks/exhaustive-deps */
 import React, { useEffect, useState } from 'react';
```

Comment:
Why don't we remove all unused occurrences of React?

Reply:
It's planned to be done in the separate PR.